testing/manual.csv

```diff
@@ -1 +1,82 @@
-"ID","Status","Comment"
+"Index","ID","Status","Comment","Assertion"
```

Comment:
In general, this looks good, but we should also consider a interop test by two independent implementations since this is a n important aspect of the profile.  Maybe this should be considered in the manual assertions?